src/TinyUPnP.h

```diff
@@ -112,6 +112,10 @@ typedef struct _ssdpDeviceNode {
 } ssdpDeviceNode;
 
 enum portMappingResult {
+    NOT_YET_ATTEMPTED, // Do not have your default value be the one you're checking for. 
```

Comment:
please remove the comment, also consider a name such as UNKNOWN, but I am fine with both.